fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68431,6 +68440,82 @@ synonym: "mVAC" EXACT []
 is_a: FBbt:00040007 ! synaptic neuropil domain
 relationship: part_of FBbt:00111027 ! adult thoracic neuromere
 
+[Term]
+id: FBbt:00047183
+name: extra l-LNv neuron
+def: "One neuron per cluster of four l-LNv neurons characterized by its more restricted projection pattern on the surface of the ipsi- and contralateral medulla." [FlyBase:FBrf0238313]
+synonym: "l-LNv extra" EXACT []
+synonym: "l-LNvx" EXACT []
+is_a: FBbt:00003763 ! l-LNv neuron
+
+[Term]
+id: FBbt:00047185
+name: ITP and CRY expressing LNd neuron
+def: "LNd neuron of the adult brain characterized by the expression of ion transport peptide (ITP) and cryptochrome (CRY). From the cell body in the anterior ventrolateral protocerebrum (AVLP), the neuron projects medially around the anterior optic tract (AOT) and dorsally along the surface of the lateral horn (LH). At the posterior surface of the LH, the neuron branches. One main branch projects towards the medulla, via the posterior lateral protocerebrum (PLP) and the accessory medulla (AME), with extensive further branching. The other main branch innervates the superior neuropils. Projections from both hemispheres overlap in the superior medial protocerebrum (SMP). This describes one of the six LNd neurons per hemisphere." [FlyBase:FBrf0238313]
+comment: Possible equivalence with 'extra LNd neuron', as both have a large cell body compared to other LNds (FlyBase:FBrf0238313 and FlyBase:FBrf0191052).
+synonym: "ITP and CRY positive LNd neuron" EXACT []
+synonym: "ITP expressing LNd neuron" EXACT []
+synonym: "ITP positive LNd neuron" EXACT []
+synonym: "ITP+ CRY+ LNd neuron" EXACT []
+synonym: "ITP+ LNd neuron" EXACT []
+synonym: "ITP+/CRY+ LNd neuron" EXACT []
+synonym: "ITP/CRY coexpressing LNd neuron" EXACT []
+is_a: FBbt:00004045 ! LNd neuron
+relationship: connected_to FBbt:00040042 ! posterior ventrolateral protocerebrum
```

Comment:
Why connected_to ?  This relationship is quite abstract and has not been used very much up to now.  Not sure it makes sense for a neuron.

Reply:
Good point. I still think best to remove.  I'm not sure the meaning will be clear to users.  Also - this interpretation doesn't quite fit with the intention of the relation.  A 'fiat boundary' is a one we declare by *fiat* that has no physical correlate. The PVLP boundary might be considered fiat, but the boundary of the neuron projection is its cell membrane which is not.  One use case for this relationship between tracts/nerves and neuropils.  Here there is typically a fiat boundary marking the end of the tract and the beginning of the neuropil that cuts through the neuron projections in the tract.  Another use case is where a tract splits and each branch gets a new name.  I think these two uses are reasonably intuitive to users.